src/Aquifer.Data/Aquifer.Data.csproj

```diff
@@ -6,6 +6,7 @@
 
     <ItemGroup>
         <PackageReference Include="Dapper" />
+        <PackageReference Include="Microsoft.Data.SqlClient" />
```

Comment:
This was a transient dependency of EF that I made explicit in order to update to the latest version.